editor/sidebar/post-excerpt/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import PanelBody from 'components/panel/body';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+import { getEditedPostExcerpt } from '../../selectors';
+import { editPost } from '../../actions';
+
+function PostExcerpt( { excerpt, onUpdateExcerpt } ) {
+	const onChange = ( event ) => onUpdateExcerpt( event.target.value );
+
+	return (
+		<PanelBody title={ __( 'Excerpt' ) } initialOpen={ false }>
+			<textarea
+				className="editor-post-excerpt__textarea"
+				onChange={ onChange }
+				value={ excerpt }
+				placeholder={ __( 'Write an excerpt (optional)' ) }
+				aria-label={ __( 'Excerpt' ) }
+			/>
+			<a href="https://codex.wordpress.org/Excerpt" target="_blank">
+				{ __( 'Learn more about manual excerpts' ) }
+			</a>
+		</PanelBody>
+	);
+	/* eslint-enable jsx-a11y/label-has-for */
+}
+
+export default connect(
+	( state ) => ( {
+		excerpt: getEditedPostExcerpt( state ),
+	} ),
+	( dispatch ) => {
+		return {
```

Comment:
I don't feel strongly. I personally prefer the compact version, but understand that it can be a bit more difficult to interpret (largely because of the need to wrap parentheses for an implicit object return value).